crates/turborepo-lib/src/commands/telemetry.rs

```diff
@@ -0,0 +1,71 @@
+use turborepo_telemetry::config::TelemetryConfig;
+use turborepo_ui::{BOLD, BOLD_GREEN, BOLD_RED};
+
+use super::CommandBase;
+use crate::cli::TelemetryCommand;
+
+fn log_status(config: TelemetryConfig, base: &CommandBase) {
+    let status = config.is_enabled();
+    match status {
+        true => {
+            println!(
+                "\nStatus: {}",
+                base.ui.apply(BOLD_GREEN.apply_to("Enabled"))
+            );
+            println!("\nTurborepo telemetry is completely anonymous. Thank you for participating!");
+        }
+        false => {
+            println!("\nStatus: {}", base.ui.apply(BOLD_RED.apply_to("Disabled")));
+            println!(
+                "\nYou have opted-out of Turborepo anonymous telemetry. No data will be collected \
+                 from your machine."
+            );
+        }
+    }
+    println!("Learn more: https://turbo.build/repo/docs/telemetry");
+}
+
+fn log_error(message: &str, error: &str, base: &CommandBase) {
+    println!(
+        "{}: {}",
+        base.ui.apply(BOLD_RED.apply_to(message)),
+        base.ui.apply(BOLD_RED.apply_to(error.to_string()))
+    );
```

Comment:
you can use the `color!` macro to make this a little cleaner
```suggestion
    println!(
        "{}: {}",
        color!(base.ui, BOLD_RED, message),
        color!(base.ui, BOLD_RED, error)
    );
```

Reply:
Ah nice, TIL